package-lock.json

```diff
@@ -7584,6 +7659,18 @@
             "integrity": "sha512-OO0pH2lK6a0hZnAdau5ItzHPI6pUlvI7jMVnxUQRtw4owF2wk8lOSabtGDCTP4Ggrg2MbGnWO9X8K1t4+fGMDw==",
             "extraneous": true
         },
+        "node_modules/ganache/node_modules/fsevents": {
+            "version": "2.3.2",
+            "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-2.3.2.tgz",
+            "integrity": "sha512-xiqMQR4xAeHTuB9uWm+fFRcIOgKBMiOBP+eXiyT7jsgVCq1bkVygt00oASowB7EdtpOHaaPgKt812P9ab+DDKA==",
+            "extraneous": true,
+            "os": [
+                "darwin"
```

Comment:
i think the `npm i` step in github actions failed because the `fsevents` lib here only supported darwin based os, but the github actions runs on linux based machine. so, i guess we need to add linux to this array too? I'll try.